docs/docs/tutorial-yaml/configuration.md

```diff
@@ -76,6 +78,10 @@ vars:
   
     # a dedicated directory for all sources
     github: "all_sources/github.yml"
+
+  _blacklist:
+    - "_FIVETRAN_SYNCED"
+    - ".*__key__.namespace"
```

Comment:
I use firestore as a data source, but there are cases where I want to ignore columns that store this kind of metadata because they are loaded into BigQuery. That's why I took up the example of using regular expressions